lib/fluent/plugin/filter_viaq_data_model.rb

```diff
@@ -15,8 +15,39 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+require 'time'
+require 'date'
+
 require 'fluent/filter'
 require 'fluent/log'
+require 'fluent/match'
+
+begin
+  ViaqMatchClass = Fluent::Match
+rescue
+  # Fluent::Match not provided with 0.14
+  class ViaqMatchClass
+    def initialize(pattern_str, unused)
+      patterns = pattern_str.split(/\s+/).map {|str|
+        Fluent::MatchPattern.create(str)
+      }
+      if patterns.length == 1
+        @pattern = patterns[0]
+      else
+        @pattern = Fluent::OrMatchPattern.new(patterns)
+      end
+    end
+    def match(tag)
+      if @pattern.match(tag)
```

Comment:
Looking at 'match' i think you could just return the result of `@pattern.match(tag)`